manifests/balancer.pp

```diff
@@ -43,10 +43,10 @@
 #   apache::balancer { 'puppet00': }
 #
 define apache::balancer (
-  Hash $proxy_set = {},
+  Hash $proxy_set           = {},
```

Comment:
I'd say we shouldn't introduce alignment. With data types it only makes things very messy. It creates a lot of merge conflicts, even for trivial PRs when a change is made.

Reply:
I don't care, which Style.  But the Style should be consistent throughout the Module.  
So, what's your Suggestion?

Reply:
I agree it should be consistent and only later noticed that other files do appear to use alignment. I'd start with a PR that only does this: stylistic changes.

@david22swan thoughts on which style to use? Perhaps that's also something all puppetlabs modules should adopt. Maybe even adopt in https://github.com/puppetlabs/language-style-guide as a recommended way for all module authors.

Reply:
From personal experience I would say I tend towards aligning variables, especially when there are a larger amount of variables, it was something that I was taught early on in Puppet and have always found it easier to read.

Reply:
As to being part of the style guide, it actually is:
https://github.com/puppetlabs/language-style-guide/blob/main/style_guide.md#parameter-indentation-and-alignment
With the instruction being that the equals signs should be aligned.

Reply:
I throw into the Pit: `pdk validate`,  which calls `puppet-lint`, which implements the Style-Guide.

Reply:
It's actually something I disagree with. Once I get back from leave I'll see about raising this in the chats we've been having about updating the style guide.